packages/core/src/components/Panel/Panel.stories.tsx

```diff
@@ -1,8 +1,10 @@
 import styled from "@emotion/styled";
 import { Close, Edit } from "@hitachivantara/uikit-react-icons";
 import { Meta, StoryObj } from "@storybook/react";
-import { HvButton, HvTypography, theme } from "index";
+import { HvButton } from "components/Button";
```

Comment:
Why not `import { HvButton, HvTypography, theme } from "components";`?

Reply:
Updated to import from `components`